stable/node-exporter/templates/daemonset.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: {{template "fullname" .}}
+  labels:
+    app: {{ template "fullname" . }}
+    heritage: "{{ .Release.Service }}"
+    release: "{{ .Release.Name }}"
+    chart: "{{.Chart.Name}}-{{.Chart.Version}}"
+spec:
+  template:
+    metadata:
+      labels:
+        app: {{template "fullname" .}}
+        release: {{.Release.Name | quote }}
+      annotations:
+        "prometheus.io/port": "9100"
+        "prometheus.io/scrape": "true"
+    spec:
+      hostNetwork: True
+      nodeSelector:
+{{ toYaml .Values.nodeSelector | indent 10 }}
+      containers:
+      - name: {{ template "fullname" . }}
+        image: "{{default "quay.io/prometheus/node-exporter" .Values.image}}:{{default "v0.13.0" .Values.imageTag}}"
```

Comment:
Don't use `default` function if not necessary. Defaults should go into `values.yaml`.